test/docker-e2e/dockerchain/config.go

```diff
@@ -0,0 +1,90 @@
+package dockerchain
+
+import (
+	"github.com/celestiaorg/celestia-app/v5/test/util/genesis"
+	"github.com/celestiaorg/celestia-app/v5/test/util/testnode"
+	"github.com/moby/moby/client"
+	"os"
+)
+
+const (
+	multiplexerImage   = "ghcr.io/celestiaorg/celestia-app"
+	defaultCelestiaTag = "v4.0.7-mocha"
```

Comment:
[follow up] this is definitely fine for now but it may be nice to automatically bump this version tag as we cut more releases. Currently there are a lot of hard-coded v3.x versions in the codebase which is a maintenance burden when we're cutting releases so I'm worried about this hard-coded version becoming similar.